src/Bernard/Driver/FlatFileDriver.php

```diff
@@ -18,13 +18,20 @@ class FlatFileDriver implements Driver
     private $baseDirectory;
 
     /**
+     * @var int
+     */
+    private $permissions;
+
+    /**
      * Constructor
      *
      * @param string $baseDirectory The base directory
+     * @param int    $permissions   Permissions to create the file with.
      */
-    public function __construct($baseDirectory)
+    public function __construct($baseDirectory, $permissions = 0)
```

Comment:
I thought about this, and considered that users _may_ have set permissions via:
https://secure.php.net/umask

If I introduce a new default here, it _may_ break other applications. 

So, either we must set an acceptable default `0760` or something, or set permissions only when explicitly provided by the user.